packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -179,7 +189,48 @@ export default class PackMoveOffersCtrl {
     set(offer, 'displayedPrice', displayedPrice);
   }
 
+  static isChosen(option) {
+    return option.choosedValue > 0;
+  }
+
   selectOffer(offer) {
-    this.$scope.$emit('offerSelected', offer);
+    const selectedOffer = offer;
+
+    const params = {
+      eligibilityReference: selectedOffer.eligibilityReference,
+      offerName: selectedOffer.offerName,
+    };
+
+    // Retrieve option values (included and added)
+    const options = selectedOffer.options
+      .filter(
+        (option) => option.included > 0 || this.constructor.isChosen(option),
+      )
+      .map((option) => ({
+        quantity:
+          option.included +
+          (this.constructor.isChosen(option) ? option.choosedValue : 0),
+        name: option.name,
+      }));
+
+    if (options.length > 0) {
+      params.options = options;
+    }
+
+    return this.OvhApiPackXdslMove.v6()
+      .servicesToDelete({ packName: this.packName }, params)
+      .$promise.then((result) => {
+        selectedOffer.subServicesToDelete = result;
+        this.$scope.$emit('offerSelected', selectedOffer);
```

Comment:
Ok as a hotfix but we should avoid `$scope` usage

Reply:
@marie-j OK what would you suggest ?